test/lib/specHelperSpec.js

```diff
@@ -49,6 +49,7 @@ class TestComponent  extends React.Component {
         <h1>Test Component</h1>
         <div className='answer'>Form Value: {this.state.value}</div>
         <input ref='value' onChange={this.getVal.bind(this)} defaultValue={this.state.value} />
+        <input id='checkboxInput' type='checkbox' ref='checkValue' onChange={this.getVal.bind(this)} />
```

Comment:
@maneeshnycda Despite our preferences in React for camelCase, we generally use hyphens in CSS identifiers.